src/index.js

```diff
@@ -122,16 +133,18 @@ module.exports = {
     'no-script-url': 'error',
     'no-self-compare': 'error',
     'no-sequences': 'error',
-    'no-shadow-restricted-names': 'error',
     'no-shadow': 'off',
-    'no-spaced-func': 'off',
+    'no-shadow-restricted-names': 'error',
+    'no-spaced-func': 'error',
     'no-sync': 'error',
-    'no-this-before-super': 'error',
```

Comment:
So this isn't related to the jscs rules, right? Just about deleting some duplication? If so, I'd suggest moving that work into a separate commit, to make each commit clearer on what it's doing. Not a blocker to get this PR merged, though, just a suggestion if you agree and have the time to do it.